tools/jslint.js

```diff
@@ -215,10 +215,10 @@ if (cluster.isMaster) {
     const elapsed = process.hrtime(startTime)[0];
     const mins = padString(Math.floor(elapsed / 60), 2, '0');
     const secs = padString(elapsed % 60, 2, '0');
-    const passed = padString(successes, 6, ' ');
-    const failed = padString(failures, 6, ' ');
+    const passed = padString(successes, 4, ' ');
+    const failed = padString(failures, 4, ' ');
```

Comment:
FWIW the reason these were larger is because of the original problem I ran into when creating jslint.js, which is that the `benchmark/tmp` directory had tens of thousands of files (~50,000 IIRC) in it (created by the module benchmark). If this or similar cases should occur, there would need to be enough space to account for such things.